nengo_gui/components/spa_similarity.py

```diff
@@ -0,0 +1,102 @@
+from .pointer import Pointer
+from nengo_gui.components.component import Component
+
+import numpy as np
+import nengo
+
+import struct
+import copy
+
+class SpaSimilarity(Pointer):
+
+    config_defaults = dict(max_value=1,
+                           min_value=-1,
+                           show_pairs=False,
+                           **Component.config_defaults)
+
+    def __init__(self, obj, **kwargs):
+        super(SpaSimilarity, self).__init__(obj, **kwargs)
+        try:
+            target_key = kwargs['target']
+        except KeyError:
+            target_key = kwargs['args']
```

Comment:
`target_key = kwargs.get('target', kwargs['args'])` would save three lines (and is potentially faster)